servers/quarkus-server/src/main/java/org/projectnessie/server/providers/InMemVersionStoreFactory.java

```diff
@@ -30,14 +32,17 @@
 @StoreType(INMEMORY)
 @Dependent
 public class InMemVersionStoreFactory implements VersionStoreFactory {
+  @Inject QuarkusDatabaseAdapterConfig config;
+
   @Override
   public <VALUE, METADATA, VALUE_TYPE extends Enum<VALUE_TYPE>>
       VersionStore<VALUE, METADATA, VALUE_TYPE> newStore(
           StoreWorker<VALUE, METADATA, VALUE_TYPE> worker, ServerConfig serverConfig) {
     DatabaseAdapter databaseAdapter =
         new InmemoryDatabaseAdapterFactory()
             .newBuilder()
-            .configure(c -> c.withConnectionProvider(new InmemoryStore()))
+            .withConfig(config)
+            .withConnector(new InmemoryStore())
```

Comment:
I think, this `InmemoryStore` instance needs to be injected (so Quarkus maintains a singleton instance).

Reply:
FIxed in 90b8280422f3dab555be727df30151ec99be8856